int16 zoom_level
int16 area_active
int16 gesture